release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -334,6 +334,28 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-5-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-5-deprecation-of-operatorsources"]
+==== OperatorSources and CatalogSourceConfigs block cluster upgrades
+
+OperatorSources and CatalogSourceConfigs have been deprecated for several
```

Comment:
`OperatorSources and CatalogSourceConfigs have been deprecated for several release` is incorrect. 

They have been `marked for deletion` for several releases, and CatalogSourceConfigs have been removed(deprecated) in 4.5. 4.4 clusters will not upgrade to 4.5 if there are any CSCs or non-default OperatorSources. 
That being said, The OperatorSource is still present in 4.5, although we still want to discourage the use of non-default OperatorSources. The OperatorSource API is only there for the default Opsrcs.

Reply:
We synced up on Slack about deprecated vs removed feature definitions per https://docs.openshift.com/container-platform/4.5/release_notes/ocp-4-5-release-notes.html#ocp-4-5-deprecated-removed-features and this should be all good now. :+1: